docs/security/trust/deploying_notary.md

```diff
@@ -0,0 +1,37 @@
+<!--[metadata]>
++++
+title = "Deploying Notary"
+description = "Deploying Notary"
+keywords = ["trust, security, notary, deployment"]
+[menu.main]
+parent= "smn_content_trust"
++++
+<![end-metadata]-->
+
+# Deploying Notary
+
+## Using Docker Compose
+
+The easiest way to deploy Notary is by using Docker Compose.
+
+Please refer to [Install Docker Compose](../../../compose/install.md) before continuing.
+
+1. Clone the Notary repository
+
+        git clone git@github.com:docker/notary.git
+
+2. Build and start Notary with the sample certificates.
+
+        docker-compose up -d
+
+
+There is more detailed documentation about how to do this at https://github.com/docker/notary.
```

Comment:
@vikstrous "this" is unclear does that refer to deploying the notary --- if so we should include all the documentation here or not include the little we do.  Do you mean running `docker-compose up`? Use a specific noun or phrase. General guidline:
![image](https://cloud.githubusercontent.com/assets/1347209/11100166/6d0d909e-8864-11e5-8771-13d6df186cb8.png)